app/lib/features/chat/providers/notifiers/chat_room_notifier.dart

```diff
@@ -227,7 +227,14 @@ class ChatRoomNotifier extends StateNotifier<ChatRoomState> {
 
   // fetch original content media for reply msg, i.e. text/image/file etc.
   Future<void> fetchOriginalContent(String originalId, String replyId) async {
-    final roomMsg = await timeline.getMessage(originalId);
+    RoomMessage roomMsg;
+    try {
+      roomMsg = await timeline.getMessage(originalId);
+    } catch (error, stack) {
+      _log.severe('Failing to load reference $replyId (from $originalId)',
+          error, stack,);
+      return;
+    }
```

Comment:
@gnunicorn Can you please comment the reason that we needn't to skip exception here?

Reply:
this is happening in a post-processing call and the exception doesn't go anywhere and thus would just be logged out anyways. and for debugging it means a lot more exceptions interjection in the process ;) . so blocking that off and logging them out was a way to clean this up a bit. but generally, we should probably do the processing differently here ...

Reply:
Sorry, I just wanted to see your comment in source code...

Reply:
I'd rather not. We should refactor that code base ... ;) .